examples/bri-3/src/bri/workgroup/workgroups/models/workgroup.ts

```diff
@@ -1,16 +1,16 @@
-import { Workstep } from '../workstep/workstep';
-import { BpiSubject } from '../../identity/bpiSubject/bpiSubject';
+import { Workstep } from '../../worksteps/models/workstep';
 import { IWorkgroup } from './workgroup.interface';
-import { Workflow } from '../workflow/workflow';
-import { Security } from '../policy/security'
-import { Privacy } from "../policy/privacy"
+import { Workflow } from '../../workflows/models/workflow';
+import { BpiSubject } from '../../../identity/bpiSubjects/models/bpiSubject';
+import { Security } from '../../../policy/models/security';
+import { Privacy } from '../../../policy/models/privacy';
 
 export class Workgroup implements IWorkgroup {
     id: string; // TODO: Add uuid after #491
     name: string;
     administrator: BpiSubject[] = []; 
-    securityPolicy: Security; //TODO Implement securityPolicy #485
-    privacyPolicy: Privacy; //TODO Implement privacyPolicy #485
+    securityPolicy: Security[]; //TODO Implement securityPolicy #485
```

Comment:
some of these fields are arrays but name is singular, for example administrator above is array, policy here is array etc, just to double check if we should use plural for names? why are these modified to arrays in this PR?

Reply:
I think we are still far from implementing these so no need to touch them atm. They will be handled as part of other stories and are here only as placeholders for now.